src/app/user/page.tsx

```diff
@@ -4,14 +4,21 @@ import { PetGrid } from '@/components/PetGrid';
 import { AddPetModal } from '@/components/modals/AddPetModal';
 import { PetData } from '@/utils/types/petData';
 import { Loader } from '@/components/Loader';
+import { removePet } from '@/data/pets/removePet';
 
 const page = () => {
   const [pets, setPets] = useState<PetData[]>([]);
-  const [petAdded, setPetAdded] = useState(false);
+  const [petAddedOrRemoved, setPetAddedOrRemoved] = useState(false);
 
```

Comment:
🟡  I believe the state name `petAddedOrRemoved` could be more descriptive. Would you consider using something like `isPetListUpdated` or `hasPetListChanged?` These alternatives might better convey the intent of tracking whether the pet list has been modified.